src/compiler/program.ts

```diff
@@ -1468,11 +1489,13 @@ namespace ts {
         }
 
         function processImportedModules(file: SourceFile, basePath: string) {
+            const maxJsNodeModuleSearchDistance = options.maxNodeModuleJsDepth || 0;

```

Comment:
so is the default is meant to be `0` or are we going to give it a higher value and users can opt in to increase it if they do not see that fit?